### ISSUE TITLE
Demeteorized application not combined or minified

### ISSUE THREAD

**Issue by chmac** [onmodulus/demeteorizer#197]
We're deploying 2 apps to modulus via the modulus npm package. For one, we first demeteorize manually, and then deploy that folder, for the other, I assume modulus does it all on its own.

In both cases, the apps are deployed, but unminified and uncombined. The javascript file for each package is served separately.

We're deploying with modulus 6.0.0 and 5.0.1. On 5.0.1, where we manually demeteorize first, we are using demeteorizer v 2.3.1. The two deploymenst are [app](http://app.trakkaapp.com.au) and [dashboard](http://trakkaapp.com.au).

Any suggestions as to what the cause is? We do have the `ecmascript` package directly added to the app project (meteor 1.3) but not for the dashboard (1.2).

**Comment by harlanj** [comment]
Hey @chmac sorry for the late response. 

1) Is there a reason why you're using `demeteorizer@2.3.1`?
2) `modulus-cli@5.0.1` uses `demeteorizer@3.x.x`. Did you manually go into the `node_modules` directory and install `demeteorizer@2.3.1`?
3) `demeteorizer` should normally minify your code when [building](https://guide.meteor.com/build-tool.html#concatenate-and-minify). What version of Meteor are you using locally?

**Comment by chmac** [comment]
Can't remember the details exactly now, but we had a problem and had to manually demeteorize first and then deploy the demeteorized directory. Something specified that version to address some specific issue.

**Comment by harlanj** [comment]
If you try to use the latest CLI and the version of `demeteorizer` installed with `modulus@5` what errors do you run into? I want to see where things go from a "clean" install to custom things needing the happen.

**Comment by chmac** [comment]
@harlanj It was over a month ago dude, don't remember what the issues were at this point. I know that we couldn't deploy with 5.0.1 directly, so we had to do the manual demeteorize step for some reason, but I don't remember the reason.

To be honest, by this point, we've decided to dump modulus and switch to something that just works, either mup(x) -> ubuntu, or galaxy. The whole modulus process and support experience was awful. We wanted to deploy an app, and I emailed support (our client is a paying customer) and was told to post an issue here. Then it takes a month to get a response. We're sailing for sunnier seas...
